Lambdas/GHHooks/EventHandler/Handlers/IssueHandler.swift

```diff
@@ -228,12 +228,10 @@ private enum Status: String {
     }
 
     init(issue: Issue) {
-        if issue.state_reason == .not_planned {
-            if issue.knownLabels.contains(.duplicate) {
-                self = .duplicate
-            } else {
-                self = .notPlanned
-            }
+        if issue.knownLabels.contains(.duplicate) {
+            self = .duplicate
+        } else if issue.state_reason == .not_planned {
+            self = .notPlanned
```

Comment:
if `duplicate` label exists then it's always a duplicate issue. Regardless if someone closed the issue as "Not Planned" or not.